content/pt/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Cluster
+id: cluster
+date: 2020-08-03
+full_link: 
+short_description: >
+   Um conjunto de servidores de processamento, também chamados de "nodes", que executam aplicações containerizadas. Todo cluster possui ao menos um "worker node" (servidor de processamento).
+   
+aka: 
+tags:
+- fundamental
+- operation
+---
+Um conjunto de servidores de processamento, chamados {{< glossary_tooltip text="nodes" term_id="node" >}}, que executam aplicações containerizadas. Todo cluster possui ao menos um "worker node" (servidor de processamento).
+
+<!--more-->
+O "worker node" hospeda os {{< glossary_tooltip text="Pods" term_id="pod" >}} que são componentes de uma aplicação. O {{< glossary_tooltip text="control plane" term_id="control-plane" >}} gerencia os "worker nodes" e os Pods no cluster. Em ambientes de produção, o "control plane" geralmente executa em múltiplos computadores e um cluster geralmente executa em múltiplos "nodes" (nós), provendo tolerância a falhas e alta disponibilidade.
```

Comment:
Is it odd to write “plano de controle”? I'd expect to see “control plane” translated. I know Italian technical writing prefers to mix English and Italian terms; I'm not sure what's normal for Portugal & Brazil.

Reply:
Yup, it's strange :D In this case I prefer keeping 'Control plane' that brings more 'power' to the meaning, than translating this :)

Reply:
When I read this term "control plane", mixed with Portuguese, it didn't look good, because it can cause some confusion for other Brazilians because they may think: is this a tool or a term? It is okay to translate this to "Plano/Planejamento de Controle", you may think that most do not know English.

Reply:
But "planejamento de controle" doesn't have the same meaning as control plane. 

Maybe "ambiente de gerenciamento" or "camada/nós de gerenciamento", WDYT?

Reply:
Morning Ricardo, I think "camada/nós de gerenciamento" than more adherent to the context, but "ambiente de gerenciamento" it's a good as too.